Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM registry.access.redhat.com/ubi9:9.0.0
```

Comment:
Doesn't the performance/ci team maintain an image with these tools already installed that we can just reuse? This way our Dockerfile would only have to COPY our scripts to it

Reply:
Not that I'm aware of, but asked in https://coreos.slack.com/archives/CNKHL6076/p1666195656425169